HISTORY.md

```diff
@@ -17,7 +17,7 @@
   * Add functions to access parameters of `Convolution` and `AtrousConvolution`
     layers (#1985).
 
-  * Add Julia bindings (#1949).  Build settings can be controlled with the
+  * Add Julia bindings (#1949). Build settings can be controlled with the
```

Comment:
This was correct before, sometimes double spaces is used to improve the readability.

Reply:
acually i just made dummy commit : )